object/verification.go

```diff
@@ -186,6 +190,24 @@ func DisableVerificationCode(dest string) {
 	}
 }
 
+func CheckSigninCode(user *User, dest, code, lang string) string {
+	// check the login error times
+	if msg := checkSigninErrorTimes(user, lang); msg != "" {
+		return msg
+	}
+
+	result := CheckVerificationCode(dest, code, lang)
```

Comment:
The result is `wrong code!`, can not go to the case `wrongCode`

Reply:
Thanks @leo220yuyaodog, I fixed it